plugins/kubernetes_events.yaml

```diff
@@ -61,15 +73,16 @@ pipeline:
         - Preempting
         - ProbeWarning
         - Rebooted
+        - Unhealthy
       error:
+        - BackOff
```

Comment:
We originally moved this from error to warning due to customer feedback. Why do we want to move it back to error?